tests/worker.test.ts

```diff
@@ -138,8 +138,7 @@ test('piscina worker test', async () => {
                 for (let i = 0; i < rounds; i++) {
                     await processCountEvents(piscina, events / batchSize, batchSize)
                 }
-                const throughput = Math.round(1000 / ((startTime - performance.now()) / events / rounds))
-                result[`${threads} thread${threads === 1 ? '' : 's'}`] = Math.round(throughput)
+                result[`${threads} thread${threads === 1 ? '' : 's'}`] = Math.round(startTime - performance.now())
```

Comment:
This is not correct as we want events per second here.

Reply:
Yep, was too fast here, fixed.